### DISCUSSION TITLE
Different ESP32 HTML for mobile and desktop response?

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#73] [Course Qeustions] [answered]
Hi Mair,

Is there something additional that need to be implemented for a mobile phone browser?

I have found my application based on the web server video works fine with Chorme or Edge browsers on desktop but when I browse to the page on my phone, the fetch GET route from the js Script does not seem to trigger any more and therefore does not call my server.c handler to update an element velue on screen like it does when using a desktop browser.

**Comment by Mair** [comment, marked as answer]
it should work the same although sometimes there are subtle differences in Safari, chrome or other browsers. have you tried a different browser?

**Reply by orbitcoms** [reply]
You are correct Mair,
Thanks for that.
I installed "Edge" browser on my phone and the page renders differently (now has white background and the updated value DOES work ok)
